README.md

```diff
@@ -33,45 +32,46 @@ Binary downloads are available here: [releases](https://github.com/IBM-Bluemix/g
 
 ## Installation
 
-No special steps are required for installation. See [Installing plug-ins in UrbanCode Deploy](https://developer.ibm.com/urbancode/docs/installing-plugins-ucd/).
+See [Installing plug-ins in UrbanCode Deploy](https://developer.ibm.com/urbancode/docs/installing-plugins-ucd/) and follow the steps for installing an automation plug-in.
 
 ## Usage
 
-### SetupBluemix 
+The IBM Globalization Pipeline plug-in for UrbanCode Deploy contains two processes that need to be configured.
+
+### SetupBluemixV2
 
 #### Description
 
-This step will look for the Globalization Pipeline service instance in your space and will create one if none existing. 
-Tf existing many instances already, the instance contains UCD in its name will take prior priority.
+This process is for configuring your Bluemix account parameters.  When configured, the Globalization Pipeline plug-in will search your Bluemix account in the organization and space that you specify for any instances of the Globalization Pipeline service.  If the Globalization Pipeline service is not found, an instance of the service will be created automatically. 
+If an instance is found that contains *UCD* in its name, that service instance will be given priority.

```

Comment:
Really minor but this would probably be:  `UCD`  (backquote) to denote quoted text